packages/backend-common/src/util/URL.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class URL {
```

Comment:
Better rely on the built-in `URL` instead tbh, skipping this extra helper. See for example https://github.com/backstage/backstage/blob/6084db33522bae49f46b05d104b5420038451087/packages/integration/src/helpers.ts#L28

Reply:
Great one thanks.